docs/content_management/field_types/field_type_reference/taxonomyentryassignmentfield.md

```diff
@@ -0,0 +1,76 @@
+# TaxonomyEntryAssignment Field Type
+
+`TaxonomyEntryAssignment` Field is used to integrate content with Taxonomy module. It allows you to select tags or categories and assign them to the content. This Field Type actually assigns tags to the content in the data action, so then you can use TaxonomyService on this content. To be able to assign tags to the content, first, you need to add a `TaxonomyEntryAssignment` Field to the Content Type definition.
+
+| Name                     | Internal name                    | Expected input |
+|--------------------------|----------------------------------|----------------|
+| `TaxonomyEntryAssignment`| `ibexa_taxonomy_entry_assignment`| array with `taxonomyEntries` and `taxonomy` keys|
+
+## PHP API Field Type 
+
+### Input expectations
+
+| Type     | Description | Example         |
+|--------|-----------------|-----------------|
+| `array` | array with `Ibexa\Contracts\Taxonomy\Value\TaxonomyEntry` objects under `taxonomy_entries` key and Taxonomy identifier under `taxonomy` key | see below |
+
+Example using `Ibexa\Taxonomy\FieldType\TaxonomyEntryAssignment\Value` object:
+``` php
+$taxonomyEntry1 = $this->taxonomyService->loadEntryByIdentifier('example_entry', 'tags');
+$taxonomyEntry2 = $this->taxonomyService->loadEntryByIdentifier('example_entry_2', 'tags');
+new \Ibexa\Taxonomy\FieldType\TaxonomyEntryAssignment\Value(
+    [
+        $taxonomyEntry1,
+        $taxonomyEntry2,
+        // ...
+    ],
+    'tags', 
+);
+```
+
+Example using array:
+``` php
+[
+    'taxonomy_entries' => [$taxonomyEntry, $taxonomyEntry2], // load entries using TaxonomyService
+    'taxonomy' => 'tags',
+]
+```
+
+### Value object
+
+#### Properties
+
+|Property|Type|Description|
+|--------|----|-----------|
+|`$taxonomyEntry`|array of `Ibexa\Contracts\Taxonomy\Value\TaxonomyEntry`|Stores selected taxonomy entry.|
```

Comment:
```suggestion
|`taxonomyEntry`|array of `Ibexa\Contracts\Taxonomy\Value\TaxonomyEntry`|Stores selected taxonomy entry.|
```
By convention we omit the `$` when mentioning properties in regular text, see for example https://doc.ibexa.co/en/master/content_management/field_types/field_type_reference/imagefield/#properties